cmd/opentelemetry/app/processor/resourceprocessor/resource_processor.go

```diff
@@ -56,6 +56,9 @@ func (f Factory) CreateDefaultConfig() configmodels.Processor {
 func (f Factory) GetTags() map[string]string {
 	tagsLegacy := flags.ParseJaegerTags(f.Viper.GetString(reporter.AgentTagsDeprecated))
 	tags := flags.ParseJaegerTags(f.Viper.GetString(resourceLabels))
+	if tags == nil {
+		tags = map[string]string{}
```

Comment:
if tags is nil can we just return `tagsLegacy` instead of going through the below logic?

Reply:
+1 done